examples/go.mod

```diff
@@ -1,10 +1,11 @@
-module github.com/getyoti/yoti-go-sdk/examples
+module github.com/getyoti/yoti-go-sdk/v2/examples
 
 go 1.12
 
 require (
-	github.com/getyoti/yoti-go-sdk/v2 v2.5.0
+	github.com/getyoti/yoti-go-sdk/v2 v2.8.0
```

Comment:
Does this need to be `2.8.0` until this is released?

Reply:
This release is 2.9.0